CHANGELOG.md

```diff
@@ -18,6 +19,8 @@
 _**Note**_: Breaking changes in the API
 * Tekton Pipeline v1beta1 *sidecars* field from a task step is now a list of `io.fabric8.tekton.pipeline.v1beta1.Sidecar` instead of a list of `io.fabric8.tekton.pipeline.v1beta1.Step`
 
+* `NetworkPolicy` moved to `io.fabric8.kubernetes.api.model.networking.v1.NetworkPolicy` from
+  `io.fabric8.kubernetes.api.model.networking.NetworkPolicy`
```

Comment:
So we no longer have access to `extensions/v1beta1` `NetworkPolicy`? Or is this provided by `V1beta1NetworkAPIGroupDSL v1beta1()` DSL method?